src/features/dashboard/components/Dialogs/DeleteAppDialog/delete-app-dialog.scss

```diff
@@ -0,0 +1,44 @@
+.modal-wrap {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: rgba(0, 0, 0, 0.5);
```

Comment:
`background: rgba(0, 0, 0, 0.5);` can we move this color to a variable @agrim-deriv ?
First check if this is already available in the color palette or not.

Reply:
Don't need this modal-wrap, removed it.